.eslintignore

```diff
@@ -0,0 +1,7 @@
+logs
+*.log
+coverage/*
+.env
+.nyc_output/*
+.vscode/*
+template/*
```

Comment:
This file ought to list files and folders eslint is to ignore and not lint it. I don't understand why you asking it to ignore files and folders that could not possibly exist in your project.